tech-cent/momo_bank_web
# #170622150 Add sign up functionality.

**What does this PR do?**
This PR adds functionality to enable users to signup (register) with momo bank 

**Description of the task to be completed?**
- setup redux
- design a signup form
- ensure that users can signup
- add feedback toasts for signup and login

**How should this be manually tested?**
- Clone and set up the project using the instruction in the README.
- $ git checkout to `ft-register-users-170622150` branch which contains the changes.
- Use $ npm start to start the application
- Click `Sign up` in the navigation bar
- Enter registration details
- Submit the details
The signup should be successful with feedback showing that you have signed up successfully

**What are the relevant pivotal tracker stories?**
[#170622150](https://www.pivotaltracker.com/story/show/170622150)

**Screenshots**
<img width="1255" alt="Screenshot 2020-01-13 at 02 32 03" src="https://user-images.githubusercontent.com/38077368/72227381-f63c1080-35ac-11ea-9ec6-c309d4cb6d9d.png">
